dbt_platform_helper/providers/copilot.py

```diff
@@ -113,27 +112,27 @@ def create_postgres_admin_task(
 
 
 def connect_to_addon_client_task(
-    ecs_client, subprocess, application_name, env, cluster_arn, task_name
+    ecs_client,
+    subprocess,
+    application_name,
+    env,
+    cluster_arn,
+    task_name,
+    addon_client_is_running_fn=get_ecs_task_arns,
```

Comment:
Not a fan of renaming things on import like this, especially when the return type I'd expect is also changing. Eg:

- I'd expect `get_ecs_task_arns` to be returning a list of something
- I'd expect `addon_client_is_running` to be returning a boolean.

Can we also rename addon_client_is_running to something like `is_addon_client_running` to make it even clearer that this is a condition?

Reply:
so thats a good catch and `addon_client_is_running` used to return a boolean where the refactor into `get_ecs_task_arns` now returns a list. So you have hit the nail on the head with naming and something that should be highlighted as a standard.